test/ui/testAppActions.js

```diff
@@ -548,6 +548,41 @@ describe('appActions', function() {
 
   });
 
+  describe('hideUnavailableDevices', function() {
+    beforeEach(function() {
+      var state = appState.getInitial();
+      app.setState(state);
+    });
+
+    describe('[windows]', function() {
+      beforeEach(function() {
+        app.setState({_os: 'win'});
+      });
+
+      it('excludes only Tandem', function() {
+        expect(app.state.uploads.length).to.equal(10);
```

Comment:
I actually love it for exactly that reason. Every time we add a device we're forced to think about whether it should be included/excluded on platforms. For one thing, should help prevent the oversight of a platform like Tandem + Windows. I'll merge as-is for now, happy to revisit.